profiler/src/main/java/com/splunk/opentelemetry/profiler/ConfigurationLogger.java

```diff
@@ -45,7 +45,7 @@ public void log(Config config) {
     log(CONFIG_KEY_INGEST_URL, config::getString);
     log(CONFIG_KEY_OTEL_OTLP_URL, config::getString);
     log(CONFIG_KEY_TLAB_ENABLED, config::getBoolean);
-    log(CONFIG_KEY_PERIOD_PREFIX + "." + "jdk.threaddump", config::getString);
+    log(CONFIG_KEY_PERIOD_PREFIX + "." + "threaddump", config::getDuration);
     logger.info("-----------------------");
```

Comment:
Actually I'm not sure at all we should be doing this. What makes profiling so special that its configuration is logged to stdout while nothing else does this?

Reply:
Would you be amenable to agreeing to this change and filing a new issue for us to discuss?